idaes/generic_models/properties/tests/test_harness.py

```diff
@@ -51,11 +52,14 @@ def configure_class(self, m):
 
         self.has_density_terms = True
 
+        self.skip_initialization_exception = False
```

Comment:
Would it be more clear if this were `skip_initialization_raises_exception_test`?